tests/MSBuildDeviceIntegration/Tests/InstallAndRunTests.cs

```diff
@@ -1075,5 +1076,42 @@ public void SupportDesugaringStaticInterfaceMethods ()
 			);
 		}
 
+		[Test]
+		public void EnableAndroidStripILAfterAOT ()
+		{
+			var proj = new XamarinAndroidApplicationProject {
+				ProjectName = nameof (EnableAndroidStripILAfterAOT),
+				RootNamespace = nameof (EnableAndroidStripILAfterAOT),
+				IsRelease = true,
+				EnableDefaultItems = true,
+			};
+			proj.SetProperty("AndroidStripILAfterAOT", "true");
+			// So we can use Mono.Cecil to open assemblies directly
+			proj.SetProperty ("AndroidEnableAssemblyCompression", "false");
+
+			var builder = CreateApkBuilder ();
+			Assert.IsTrue (builder.Build (proj), "`dotnet build` should succeed");
+
+			var apk = Path.Combine (Root, builder.ProjectDirectory, proj.OutputPath, $"{proj.PackageName}-Signed.apk");
+			FileAssert.Exists (apk);
+			var helper = new ArchiveAssemblyHelper (apk);
+			Assert.IsTrue (helper.Exists ($"assemblies/{proj.ProjectName}.dll"), $"{proj.ProjectName}.dll should exist in apk!");
+			using (var stream = helper.ReadEntry ($"assemblies/{proj.ProjectName}.dll")) {
+				stream.Position = 0;
+				using var assembly = AssemblyDefinition.ReadAssembly (stream);
+				var type = assembly.MainModule.GetType ($"{proj.RootNamespace}.MainActivity");
+				var method = type.Methods.FirstOrDefault (p => p.Name == "OnCreate");
+				Assert.IsNotNull (method, $"{proj.RootNamespace}.MainActivity.OnCreate should exist!");
+				Assert.IsTrue (!method.HasBody || method.Body.Instructions.Count == 0, $"{proj.RootNamespace}.MainActivity.OnCreate should have no body!");
```

Comment:
`HasBody` was true and `Body.Instructions.Count` was 0.

Reply:
This is probably expected, since I only zero'ed out method body and preserved method method header.